### DISCUSSION TITLE
Vote of thanks !

### DISCUSSION THREAD

**Discussion by TecnikOfficial** [RareDevs/Rare#281] [General]
thank u for making this GUI of legendary its light fast and gets the job done . i switched from epic games launcher coz it takes long time to boot plus very time launcher update comes it gets verifying update loop bug .
also i prefer rare over heroic launcher coz of its low resource impact coz in gaming each process count 
the best thing for privacy concerned people is the rare launcher doesnt even need to ask for uac prompt when installing also here r its virustotal result : https://www.virustotal.com/gui/file/c2bc591b5e4b3c270c658d3000ed9bc8c8a177904575bb44cb5d6664d341c525
although its not signed shows author unknown from quick research it seems pretty safe

**Comment by loathingKernel** [comment]
Thank you!
